### PR TITLE
Creating aveCCV2F 

### PR DESCRIPTION
**Pull request by sgkang**
Creating `aveCCV2F` for self-potential inversion problem.

### CODE REVIEW

File: discretize/DiffOperators.py [hunk 1]
```diff
@@ -774,6 +774,37 @@ def aveCC2F(self):
...
+    # TODO: modularize
+    @property
+    def aveCC2FV(self):
+        """
+        Construct the averaging operator on cell cell centers to
+        faces as a vector.
+        """
+        if getattr(self, '_aveCC2FV', None) is None:
+            if self.dim == 1:
+                self._aveCC2FV = self._aveCC2Fx
```
Comment (lheagy) [inline discretize/DiffOperators.py]: does `_aveCC2Fx` exist somewhere?

File: discretize/DiffOperators.py [hunk 2]
```diff
@@ -774,6 +774,37 @@ def aveCC2F(self):
...
+            elif self.dim == 2:
+                self._aveCC2Fx = sp.kron(speye(self.nCy), av_extrap(self.nCx))
+                self._aveCC2Fy = sp.kron(speye(self.nCy), av_extrap(self.nCx))
+                self._aveCC2FV = sp.block_diag((
+                        self.aveCC2Fx, self.aveCC2Fy
+                        ), format="csr")
+            elif self.dim == 3:
+                self._aveCC2Fx = kron3(
```
Comment (lheagy) [inline discretize/DiffOperators.py]: sorry to be picky, would you mind closing the bracket on the same level as it the `self`? eg
```
self._aveCC2Fz = kron3(
    some_things
)
```

**Comment by lheagy** [comment]
do we want to call this `aveCCV2F` (it is really a cell centered vector going to faces) thoughts @sgkang?

**Comment by lheagy** [comment]
@sgkang: please feel free to ping when you have updated the name and we can get this in!

**Comment by sgkang** [comment]
@lheagy this is ready to be merged.

**Comment by lheagy** [comment]
closes #87

**Pull request merged**